client/a8c-for-agencies/components/layout/style.scss

```diff
@@ -0,0 +1,287 @@
+	background-color: rgba(246, 247, 247, 0.95);
+	box-shadow: 2px 2px 2px 0 rgb(0 0 0 / 8%);
+	z-index: 1001;
+	height: 74px;
+
+	.a4a-layout__header {
+		flex-wrap: nowrap;
+		max-width: 1500px;
+		padding-inline: 48px;
+
+		> * {
+			width: auto;
+		}
+	}
+
+	.a4a-layout__header-main,
+	.a4a-layout__header-actions {
+		margin: 0;
+	}
+
+	.a4a-layout__header-subtitle {
+		display: none;
+	}
+
+	.a4a-layout__header-title {
+		font-size: rem(24px);
+		margin-block-end: 0;
+		display: none;
+
+		@include break-large {
+			display: block;
+		}
+	}
+
+	@include breakpoint-deprecated( ">660px" ) {
+		width: calc(100% - var(--sidebar-width-min));
+		left: var(--sidebar-width-min);
+	}
+
+	@include breakpoint-deprecated( ">960px" ) {
+		width: calc(100% - var(--sidebar-width-max));
+		left: var(--sidebar-width-max);
+	}
+}
+
+.a4a-layout__header-breadcrumb {
+	margin-block-end: 4px;
+}
+
+.a4a-layout__header-title {
+	font-size: 2.25rem;
+	margin-block-end: 8px;
+	font-weight: 600;
+	line-height: 1.2;
+}
+
+.a4a-layout__header-subtitle {
+	font-size: 1rem;
+	color: var(--studio-gray-60);
+	margin: 0;
+	font-weight: 400;
+	line-height: 1.2;
+}
+
+.section-nav.a4a-layout__navigation {
+	margin-block-start: 16px;
+
+	.section-nav__mobile-header-text .count {
+		margin-inline-start: 8px;
+	}
+
+	.select-dropdown__item.is-selected .count {
+		color: var(--color-text);
+	}
+
+	.select-dropdown__header {
+		border-width: 0;
+
+		.count {
+			top: 12.5px;
+		}
+
+		@include breakpoint-deprecated( ">660px" ) {
+			border-width: 1px;
+		}
+	}
+
+	.section-nav-tabs.is-dropdown {
+		width: 100%;
+		margin: 0 0 1px 0;
+
+		@include breakpoint-deprecated( ">660px" ) {
+			margin-block-end: 12px;
+		}
+	}
+
+	.select-dropdown__options {
+		margin-inline: -1px;
+	}
+
+	.section-nav-tabs__dropdown .select-dropdown__container {
+		max-width: unset;
+		width: 100%;
+	}
+
+	.section-nav-tabs__dropdown {
+		// Since the search below the dropdown has z-index: 22,
+		// we need to make sure the dropdown is above it
+		z-index: 23;
+	}
+
+	@include breakpoint-deprecated( ">960px" ) {
+		margin-block-start: 32px;
+	}
+}
+
+
+.a4a-layout__stepper {
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+	margin-block: 16px 32px;
+}
+
+.a4a-layout__stepper-step {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+
+	.a4a-layout__stepper-step-circle {
+		width: 20px;
+		height: 20px;
+		display: flex;
+		border-radius: 50%;
+		justify-content: center;
+		align-content: center;
+		align-items: center;
+		font-size: 0.75rem;
+	}
+
+	.a4a-layout__stepper-step-name {
+		margin-left: 0.5rem;
+		white-space: nowrap;
+	}
+
+	.a4a-layout__viewport {
+		margin-inline-start: 0;
+	}
+
+	&.is-current > .a4a-layout__stepper-step-circle {
+		background-color: var(--studio-gray-60);
+		border: 2px solid var(--studio-gray-60);
+		color: var(--studio-white);
+	}
+
+	&.is-next > .a4a-layout__stepper-step-circle {
+		border: 2px solid var(--studio-gray-60);
```

Comment:
Can we ensure we swap all --studio colors with the --color version so this gets updated when we have the color scheme for the A4A environment?

You can refer to the [list](https://github.com/Automattic/wp-calypso/blob/HEAD/packages/calypso-color-schemes/src/shared/color-schemes/_default.scss) here for equivalent variable.

Reply:
Thanks for the review. 

Good point. I have updated it now: https://github.com/Automattic/wp-calypso/pull/87629/commits/e53b1e0fafb0084ef288d8f883b3a0d8ca2de050